packages/react-dom-bindings/src/events/ReactDOMEventListener.js

```diff
@@ -319,6 +319,7 @@ export function getEventPriority(domEventName: DOMEventName): EventPriority {
     case 'ratechange':
     case 'reset':
     case 'resize':
+    case 'scrollend':
```

Comment:
Added this to `DiscreteEventPriority` rather than `ContinuousEventPriority` which would match `onScroll`. I think this is right since the event only fires once at the end of a scroll, but not fully sure about the implications.

Reply:
Not sure this sounds right. I think we use discrete for things where there’s user intent and where the user can tell the difference between “doing it once” and “doing it twice”. Should this be default pri instead?

Reply:
Ah ok. I wasn't really sure what the implications of the priorities were. Just noticed that e.g. pointerdown/pointerup were marked discrete and pointermove was marked continuous so thought it was something related to the frequency. Should I omit it here so it falls back to default below? What's the difference between discrete and default?

Reply:
ah just found this

* **Discrete events:** If the active event is _directly caused by the user_ (such as mouse and keyboard events) and _each event in a sequence is intentional_ (e.g. `click`), return `DiscreteEventPriority`. This tells React that they should interrupt any background work and cannot be batched across time.

* **Continuous events:** If the active event is _directly caused by the user_ but _the user can't distinguish between individual events in a sequence_ (e.g. `mouseover`), return `ContinuousEventPriority`. This tells React they should interrupt any background work but can be batched across time.

* **Other events / No active event:** In all other cases, return `DefaultEventPriority`. This tells React that this event is considered background work, and interactive events will be prioritized over it.